### PR TITLE
Traitor Item: PLAP - The Portable Listening Advanced Post

### PR DESCRIPTION
**Pull request by IgiariValkyr**
## About The Pull Request

Ever wanted to do a listening post? Upset that the ghost roles screwed you out of it because the one ruin that it used to work just spawns a braindead mob instead? Look no further than the PLAP!

For just ~~20~~ 10 TC, you can have a listening post wherever you like! These posts include:

- [x] A PDA Message Monitor to send out spam emails and read sexts!
- [x] An Advanced Camera System so you can glide gracefully through the station, staring at people being exceedingly obvious traitors!
- [x] Medical Records, to look up information about that one cute girl you just saw!
- [x] A fax machine to send weird cryptic threats to the station or admins!
- [x] A photocopier. It's not special. It's just a photocopier.
- [x] A FREE™ Chameleon Stamp to make all falsified paperwork official!
- [ ] Basic communication knowledge!
- [ ] Self-contained power!

_Does not actually contain communication knowledge or power systems._

___

**_In actuality though, it's a 3x3 listening post box. It must be thrown in an area with power, or it will simply not function as it runs off the power grid._**

## Why It's Good For The Game

People always seem to love Listening Post gimmicks, but Dauntless/Interdyne don't have the full spread for proper shenanigans. This allows traitors to give up TC to get a full listening post kit, and still have ~~a few~~ enough left over a small side gimmick.

## Proof Of Testing

mpc-hc64_2024-02-23_17-55-38
mpc-hc64_2024-02-23_18-02-41
ShareX_2024-02-24_03-16-46

## Changelog

:cl:
add: New Traitor Item (10TC) - Portable Listening Advanced Post
/:cl:

### CODE REVIEW

**Comment by IgiariValkyr** [comment]
Don't know why MapDiff threw a fit over a piece of paper. The paper works fine in testing both on server as an imported template, and testing for the proper pull.

**Comment by StrangeWeirdKitten** [comment]
image

Unknown constant type.

File: _maps/templates/listeningbasics.dmm [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+	pixel_x = 7;
+	pixel_y = 12
+	},
+/obj/item/pen/fourcolor{
+	pixel_x = -5;
+	pixel_y = 7
+	},
+/obj/item/paper/crumpled/fluff{
+	pixel_x = 7;
+	default_raw_text = "<h1><table bgcolor="darkred" width="100%"><th><font color="#ffffff" size=4 face="OCRAStd,monospace"><div align="center">♦♦♦♦♦♦♦♦◹§◺♦♦♦♦♦♦♦♦</font></div></th></table></h1><hr /><font face="Veranda" size=3>Welcome to your very own "Portable" Listening Post! Now that you've unboxed the kit, lets get started on how everything works!<br><br>Included in this kit are:<br>* State of the art protective windows, from the dirty outside world!<br>* A computer bay fit to make the most hackerman of boys cry!<br>* A fax machine painted in sleek Syndicate Red.<br>* A filing cabinet! It's not special. Just a filing cabinet.<br><br>Listen to your foes! Stink up their comms! Find a cute girl to stalk! All is possible with the "Portable" Listening Post!<br><hr />The fax machine included in the PLP is set up to the Sothran network. It is not visible on any fax machines aboard the station, while still able to send to them. Use this to your advantage.</font><hr><center><font size=3>❘❙❚❘❙❚|<span style="color:black;
```
Comment (Majkl-J) [inline _maps/templates/listeningbasics.dmm:102]: I'm pretty sure this is missing airquotes /somewhere/ in this mess

Comment (BurgerLUA) [reply _maps/templates/listeningbasics.dmm:102]: Everywhere. Basically just wrap the whole thing with {} and it should be good.

Comment (BurgerLUA) [reply _maps/templates/listeningbasics.dmm:102]: Example: https://github.com/Bubberstation/Bubberstation/blob/master/modular_zubbers/code/game/machinery/burger_reactor/reactor_guide.dm

**Comment by IgiariValkyr** [comment]
ok, i see what's going on. It's hitting a character limit in the dmm file. I have to make an actual paper item for it.

**Comment by IgiariValkyr** [comment]
WOOOOOOOOOO

**Comment by IgiariValkyr** [comment]
And now the linters don't like my corners? Good god. I see other stuff with those corners!

**Comment by IgiariValkyr** [comment]
Failed on metastation linter. As of reading it:
**Not my fault. Fix your monkey business code.**

**Comment by IgiariValkyr** [comment]
> Maybe make it more like 5-10 TC, there's nothing really you can't acquire normally besides the camera console. And this kind of encourages antags to just do nothing but watch the crew and slightly annoy them.

I'll adjust the cost and see if I can make it show up in the discount tabs too.

**Comment by IgiariValkyr** [comment]
Not sure if I can discount it or not because I added it to the Bundle tab, but it's at a 10TC cost now based on more feedback.

**Comment by Majkl-J** [comment]
I thought I merged this

**Comment by IgiariValkyr** [comment]
Pinging myself for this. Something in the latest pull reverted the capsule file, so the capsule for the PLAP no longer is Black and Red. Add to "minor fix PR #4"

**Pull request merged**